### ISSUE TITLE
3D mode

### ISSUE THREAD

**Issue by opm0** [bemusic/bemuse#430]
The game's performance could be a lot better in devices with slow 2D (like the Samsung ARM Chromebook) if a 3D WebGL mode was added.

**Comment by dtinth** [comment]
Yeah I’m thinking about adding a flag to enable 3d WebGL based renderer.

If possible, can you try [running a local version of Bemuse](https://github.com/bemusic/bemuse/blob/master/README.md#development) and update [scintillator/context.js](https://github.com/bemusic/bemuse/blob/7ec14efda6ceb672a31a5ecdf49f9adfc0175208/src/scintillator/context.js#L13) to use [PIXI.WebGLRenderer](http://pixijs.download/release/docs/PIXI.WebGLRenderer.html)?

**Comment by opm0** [comment]
The notes are invisible if I change PIXI.CanvasRenderer to PIXI.WebGLRenderer, so that's not very useful right now.
